utils/frame/benchmarking-cli/src/pallet/mod.rs

```diff
@@ -166,4 +166,9 @@ pub struct PalletCmd {
 	/// template for that purpose.
 	#[clap(long)]
 	pub no_storage_info: bool,
+
+	/// A path to a `.json` file with existing benchmark results. When specified the benchmarks
+	/// are not actually executed, and the data for the analysis is read from this file.
```

Comment:
Maybe mention here that these files can be generated with `--json/--json-output`.